decidim-proposals/spec/factories.rb

```diff
@@ -1,5 +1,6 @@
 require_relative "../../decidim-core/spec/factories"
 require_relative "../../decidim-admin/spec/factories"
+require_relative "../../decidim-comments/spec/factories"
```

Comment:
Unrelated to this PR, but now that I see this, looks like we should remove this relative dependencies and turn them into regular dependencies like:

```ruby
require "decidim/core/test/factories"
require "decidim/admin/test/factories"
require "decidim/comments/test/factories"
```

Relevant for the work in https://github.com/AjuntamentdeBarcelona/decidim/pull/371